Tests/KituraNetTests/LargePayloadTests.swift

```diff
@@ -39,10 +39,25 @@ class LargePayloadTests: KituraNetTest {
 
     private let delegate = TestServerDelegate()
 
+    private func uniqueTemporaryFilePath() -> String {
```

Comment:
This all looks fine, I used a slightly different implementation in the equivalent Kitura-net PR - take a look at my `setUp()` and `tearDown()` in [`Tests/KituraNetTests/UnixSocketTests.swift`](https://github.com/IBM-Swift/Kitura-net/blob/aa1755c6d88ae8dc4b9483a0dc608e95251816d5/Tests/KituraNetTests/UnixSocketTests.swift)
\- in particular, it would be nice to not leave a bunch of temporary files lying around after running the tests.

Also, is it possible for the test to assert that the client is connecting over a Unix socket? I did this in the test I referred to above, but I'm not sure if this is feasible with NIO.